### ISSUE TITLE
Parametric Models

### ISSUE THREAD

**Issue by w1806291** [goranbrostrom/eha#12]
Can you help with this code?

plot(fit.pch, fn = "haz")

Error in hpch(xx, x$cuts, score * x$hazards[i, ]) : Must be one more level than cut.

and this code?

fit.boot <- glmmboot(event ~ sex + civ + region, cluster = time,
family = binomial(link = cloglog),
data = om3)

Error in glmmboot(event ~ sex + civ + region, cluster = time, family = binomial(link = cloglog),  : 
  could not find function "glmmboot"

Thanks,

**Comment by goranbrostrom** [comment]
No, I cannot, in neither case. I suggest that you contact the authors of the relevant packages, 'eha' in the first case (I guess), and 'glmmML' in the second case. Don't forget to provide reproducible examples and to identify yourself. This is not the place for tutorials. But in the case of trying examples from the book "Event History Analysis with R" I can recommend looking at my github repository EHAR2, where the forthcoming Second Edition of the book resides. Clone it and you will get all the relevant R code you were asking for. And use the latest version of eha.

Good luck, Göran
